profiles/views_test.py

```diff
@@ -0,0 +1,136 @@
+"""
+Tests for profile view
+"""
+import json
+
+from dateutil.parser import parse
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from rest_framework.status import (
+    HTTP_403_FORBIDDEN,
+    HTTP_405_METHOD_NOT_ALLOWED,
+)
+
+from profiles.factories import ProfileFactory
+from profiles.models import Profile
+from profiles.serializers import (
+    ProfileLimitedSerializer,
+    ProfilePrivateSerializer,
+    ProfileSerializer,
+)
+
+
+class ProfileTests(TestCase):
+    """
+    Tests for GET on profile view
+    """
+
+    def setUp(self):
+        """
+        Create a user and profile
+        """
+        super(ProfileTests, self).setUp()
+
+        self.user1_credentials = {
```

Comment:
We don't actually need to store these anymore. Users can be forcibly logged in with Django 1.9's `force_login` function. Additionally, `force_login` skips some of the authorization checks around login, which makes it faster. https://docs.djangoproject.com/en/1.9/topics/testing/tools/#django.test.Client.force_login